internal/cli/internal/command/services/postgres/connect.go

```diff
@@ -0,0 +1,93 @@
+package postgres
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/superfly/flyctl/internal/cli/internal/command"
+	"github.com/superfly/flyctl/internal/cli/internal/command/ssh"
+
+	"github.com/superfly/flyctl/internal/cli/internal/flag"
+	"github.com/superfly/flyctl/internal/client"
+	"github.com/superfly/flyctl/pkg/agent"
+)
+
+func newConnect() (cmd *cobra.Command) {
+	const (
+		long = `
+			Connect to the Postgres console
+		`
+		short = "Connect to the Postgres console"
+		usage = "connect [APPNAME]"
+	)
+
+	cmd = command.New(usage, short, long, runConnect,
+		command.RequireSession,
+	)
+	cmd.Args = cobra.MaximumNArgs(1)
+
+	flag.Add(cmd,
+		flag.String{
+			Name:        "database",
+			Shorthand:   "d",
+			Description: "The name of the database you would like to connect to",
+			Default:     "postgres",
+		},
+		flag.String{
+			Name:        "user",
+			Shorthand:   "u",
+			Description: "The postgres user to connect with",
+			Default:     "postgres",
+		},
+		flag.String{
+			Name:        "password",
+			Shorthand:   "p",
+			Description: "The postgres user password",
+		},
+	)
+
+	return
+}
+
+func runConnect(ctx context.Context) error {
+	client := client.FromContext(ctx).API()
+
+	appName := flag.FirstArg(ctx)
```

Comment:
If you have a command which expects an app name, we've built helpers for that. You can check how `apps open` behaves, as an example, [here](https://github.com/superfly/flyctl/blob/79ff1a15d3d3eef7b15b27a084fb84ad6679cbe0/internal/cli/internal/command/apps/open.go).